app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  before_action :current_user, only: [:create]
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.author_id = current_user.id
+    @comment.post_id = params[:post_id]
+
+    if @comment.save
+      redirect_to user_post_path(current_user.id, Post.find(params[:post_id]))
+    else
+      render :new
+    end
+  end

```

Comment:
- [ ] Nice job for including the `new `and `create` actions that can enable the current login user to add a comment for a post unfortunately, there's no form in views that accepts inputs for creating comments. Please can we consider adding a form for this aspect?. Also, remember the requirements demand us to have a form for adding comments.

Reply:
Thanks @FrederickMih for the review. I have included the comment form in `posts/:id/show` to make a comment on the  post and Below is the link demonstrating this feature.

[Demo video ](https://www.loom.com/share/9271109c323549c4b42d6a90f12a9f52)